api4/user_test.go

```diff
@@ -261,6 +261,39 @@ func TestGetUserByEmail(t *testing.T) {
 	}
 }
 
+func TestGetProfileImage(t *testing.T) {
+	th := Setup().InitBasic().InitSystemAdmin()
+	defer TearDown()
+	Client := th.Client
+	user := th.BasicUser
+
+	data, resp := Client.GetProfileImage(user.Id, "")
+	CheckNoError(t, resp)
+	if data == nil || len(data) == 0 {
+		t.Fatal("Should not be empty")
+	}
+
+	_, resp = Client.GetProfileImage(user.Id, resp.Etag)
+	if resp.StatusCode != http.StatusNotModified {
+		t.Fatal("Should have hit etag")
+	}
```

Comment:
I should have implemented like below but I couldn't get it right. Let me know your thought on this. thanks

```
data, resp = Client.GetProfileImage(user.Id, resp.Etag)
CheckEtag(t, data, resp)
```